### DISCUSSION TITLE
Removing the -X parameter in curl and adding --location --request when generating code.

### DISCUSSION THREAD

**Discussion by sln13** [swagger-api/swagger-codegen#11910] [Q&A]
Hi, Newbie in using swagger codegen here.
So, when I use swagger codegen to generate client SDK, it seems like the parameter '-X' is always used. So that becomes a problem when there are redirects in the request. So wondering how to remove that and add other parameters like --location and --request.

**Comment by fdv0695** [comment]
I'm interested
